src/azul/plugins/repository/tdr/__init__.py

```diff
@@ -241,10 +241,12 @@ def _stitch_bundles(self,
                 unprocessed |= self._find_upstream_bundles(dangling_inputs) - processed
             else:
                 log.debug('Bundle %r is self-contained', bundle)
-        if log.isEnabledFor(logging.DEBUG):
-            log.debug('Stitched together bundles: %r', processed)
-        else:
-            log.info('Stitched together %i bundles', len(processed))
+        processed.remove(root_bundle)
+        if processed:
+            msg = f'Stitched {len(processed)} bundles'
```

Comment:
```suggestion
            msg = f'Stitched {len(processed)} bundle(s)'
```

Reply:
I think that's an improvement. What do you think about 
```
arg = f': {processed!r}' if log.isEnabledFor(logging.DEBUG) else ''
log.info('Stitched %i bundle(s)%s', len(processed), arg)
``` 
to preserve the `%r` behavior?

Reply:
Yes, let's go with that.